### PR TITLE
UIU-2326 - create Jest/RTL test for DepartmentsNameEdit.js

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
Issue : https://issues.folio.org/browse/UIU-2326
create Jest/RTL test for DepartmentsNameEdit.js

### CODE REVIEW

File: src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+const onSubmit = jest.fn();
+
+const arrayMutators = {
+  pop: jest.fn(),
+  push: jest.fn(),
+};
+
+const renderDepartmentsNameEdit = (props) => {
+  const component = () => (
+    <>
```
Comment (BogdanDenis) [inline src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js:24]: No need for fragments here, we can just do `component = () => <DepartmentsNameEdit {...props} />`

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js (hunk 1 above)
Comment (Terala-Priyanka) [reply src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js:24]: Thank you, I will remove it.

File: package.json [hunk 2]
```diff
@@ -921,6 +921,7 @@
     "@folio/plugin-find-user": "^6.0.0"
   },
   "resolutions": {
-    "moment": "~2.24.0"
+    "moment": "~2.24.0",
+    "@vue/compiler-sfc": "3.2.33"
```
Comment (mkuklis) [inline package.json:925]: @Terala-Priyanka this should not be required after https://github.com/folio-org/ui-users/pull/2144 is merged

Comment (Terala-Priyanka) [reply package.json:925]: @mkuklis 
Thank you for the inputs. Shall I put this PR in draft mode till #2144 is merge?

Comment (Terala-Priyanka) [reply package.json]: @mkuklis I have removed the dependency resolution from package.json in the interest of #2144

**Review by BogdanDenis — APPROVED** [review]

File: src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  it('should render "Department name"', () => {
+    expect(screen.getByText('ui-users.extended.department.name')).toBeInTheDocument();
+  });
+
+  it('should render "Add Department" button', () => {
+    expect(screen.getByText('ui-users.extended.department.add')).toBeInTheDocument();
+  });
+
+  it('should render select compoenent after clicking "Add Department" button', () => {
```
Comment (zburke) [inline src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js:61]: test code looks great; minor typo in the caption `compoenent` should be `component`.

**Review by zburke — CHANGES_REQUESTED** [review]
Please fix the minor typo then LGTM!

File: src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js (hunk 3 above)
Comment (Terala-Priyanka) [reply src/components/EditSections/EditExtendedInfo/DepartmentsNameEdit/DepartmentsNameEdit.test.js]: @zburke  Corrected the typo.
Thank you!

**Pull request merged**